pytest.ini

```diff
@@ -9,7 +9,6 @@ log_cli_level = INFO
 testpaths =
     tests
 markers =
-    skipifroxar: skipped if in roxar environment
-    skipunlessroxar: skipped if not in the roxar environment
     bigtest: A big test not executed by default
```

Comment:
Is bigtest needed now?

Reply:
There are still some tests that we mark as taking a long time, but all the non-marked tests take less than 5 seconds now.